diagnostic.md

```diff
@@ -57,13 +63,19 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li').find('body').hide()
```

Comment:
Not quite. You first need to navigate up to parent, like this `$('li').parent().parent().find('form');` or this `$('li').parent().parent().children('form');`